iodata/formats/mwfn.py

```diff
@@ -0,0 +1,367 @@
+# IODATA is an input and output module for quantum chemistry.
+# Copyright (C) 2011-2019 The IODATA Development Team
+#
+# This file is part of IODATA.
+#
+# IODATA is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# IODATA is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <http://www.gnu.org/licenses/>
+# --
+"""Multiwfn MWFN file format."""
+
+from typing import Tuple
+
+import numpy as np
+import scipy.constants as spc
+
+from ..basis import HORTON2_CONVENTIONS, MolecularBasis, Shell
+from ..docstrings import document_load_one
+from ..orbitals import MolecularOrbitals
+from ..utils import LineIterator
+
+__all__ = ['load_mwfn_low']
+
+PATTERNS = ['*.mwfn']
+
+# From the MWFN chemrxiv paper
+# https://chemrxiv.org/articles/Mwfn_A_Strict_Concise_and_Extensible_Format
+# _for_Electronic_Wavefunction_Storage_and_Exchange/11872524
+# For cartesian shells
+# S shell: S
+# P shell: X, Y, Z
+# D shell: XX, YY, ZZ, XY, XZ, YZ
+# F shell: XXX, YYY, ZZZ, XYY, XXY, XXZ, XZZ, YZZ, YYZ, XYZ
+# G shell: ZZZZ, YZZZ, YYZZ, YYYZ, YYYY, XZZZ, XYZZ, XYYZ, XYYY, XXZZ, XXYZ, XXYY, XXXZ, XXXY, XXXX
+# H shell: ZZZZZ, YZZZZ, YYZZZ, YYYZZ, YYYYZ, YYYYY, XZZZZ, XYZZZ, XYYZZ, XYYYZ, XYYYY, XXZZZ,
+#           XXYZZ, XXYYZ, XXYYY, XXXZZ, XXXYZ, XXXYY, XXXXZ, XXXXY, XXXXX
```

Comment:
These differ from the HORTON2 conventions. Can you add them explicitly to the `CONVENTIONS` list below?

Reply:
will do.